utils/motorcontroller/MotorConfig.java

```diff
@@ -13,7 +13,7 @@ public class MotorConfig {
 
     public enum Motor_Type {
 
-        NEO, NEO_550, REDLINE_775PRO, BAG, CIM;
+        NEO, NEO_550, REDLINE_775PRO, BAG, CIM, FALCON_500, KRAKEN_X60;
```

Comment:
Interesting that we call it KRAKEN_X60 here and KRAKEN in the MotorType.java file. This seems like the right way here

And we don't need FALCON_500 - we'll never use those motors